gpu_cluster/controllers/gpu_container_controller.py

```diff
@@ -51,11 +52,12 @@ def create_container(self, image, user="", token_required=False, budget=-1, num_
         token = ""
         if token_required: 
             token = self.docker_client.exec_run(c_id, 'python3 /opt/cluster-container/jupyter_get.py')
-            uurl = "http://vault.acm.illinois.edu:{}/?token={}".format(uport, token.decode("utf-8") )
-            murl = "http://vault.acm.illinois.edu:" + str(mport)
+            base_url = "http://{}".format(self.config["domain_name"])
```

Comment:
also add support for this in the CPU controller

Reply:
Done @narendasan